code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1019,7 +1019,10 @@
 
 /mob/living/carbon/human/species/lizard
 	race = /datum/species/lizard
-
+// yogs start
+/mob/living/carbon/human/species/gorilla
+	race = /datum/species/gorilla
+// yogs end

```

Comment:
Can you dump this class definition into ``/yogstation/code/modules/mob/living/carbon/human/human.dm``?